@ecomplus/storefront-components/src/scss/TheCart.scss

```
@@ -0,0 +1,55 @@
+.cart {
+  &__empty {
+    width: 100%;
+    text-align: center;
+  }
+
+  .cart-item {
+    padding-top: 1rem;
+    border-top: 1px dotted var(--gray);
+
+    &__name {
+      margin-bottom: .5rem;
+    }
+  }
+
+  &__discount {
+    margin: 1.5rem 0 1rem;
+
+    @media (max-width: 575.98px) {
+      margin: .5rem 0;
+    }
+  }
```

Nos componentes novos esta lógica é invertida também, mobile first sempre que possível, então seria:

```suggestion
  &__discount {
    margin: .5rem 0;

    @media (min-width: 576px) {
      margin: 1.5rem 0 1rem;
    }
  }
```